NSURL.m

```diff
@@ -156,6 +157,19 @@ static void posixError(CFErrorRef *error) {
     CFRelease(err);
 }
 
+static void dlError(CFErrorRef *error){
```

Comment:
```suggestion
static void dlError(CFErrorRef *error) {
```

Reply:
should I fix this for `posixError` as well?

Reply:
Ah, I haven't looked at that, but sure, please fix that one too.